wiki/Guides/Changing_the_Artist_or_Title/id.md

```diff
@@ -0,0 +1,29 @@
+# Cara mengganti artis atau judul lagu dari sebuah beatmap
+
+*[Full Submission with Title/Artist Change -- Mistakes oleh NoHitter (dibajak oleh peppy)](https://osu.ppy.sh/community/forums/topics/24443)*
+
+Ketika mengganti **judul lagu atau artis** sebuah map, kamu dapat menemui masalah di mana kamu tidak dapat memperbarui map tersebut.
+
+**Jangan mengirim map ketika menu "Web" pada editor ditutup. Map kamu akan menjadi duplikat.**
+
+Untuk memperbaiki map yang terduplikasi, lakukan hal berikut ini:
+
+- Metode mudah (beatmap tidak memiliki perubahan pada lokal):
+  1. Hapus map yang berada pada direktori lokalmu.
+  2. *Download* map tersebut dari situs web osu!.
```

Comment:
unduh?

Reply:
"Download" sounds better for this kind of thing, so it better be left there, also it has been italicized, so it's pretty much fine ^^